### ISSUE TITLE
DFHACK_HEADLESS will Segmentation fault

### ISSUE THREAD

**Issue by ralpha** [DFHack/dfhack#1589]
Tested on Linux:
When setting `DFHACK_HEADLESS` dfhack will always crash on startup with a `Segmentation fault (core dumped)`
```bash
$ export DFHACK_HEADLESS=1
$ ./dfhack
Loading bindings from data/init/interface.txt
Resetting textures
Segmentation fault (core dumped)

$
```
Tested with 0.47.04-r1 (release) on x86_64 Linux (self compiled although works normally when doing anything else)
(`dfhack.init` == `dfhack.init-example`)

**Comment by lethosor** [comment]
Pretty sure I fixed this in https://github.com/DFHack/dfhack/commit/24cebbda5813450b0196253cc7c26aa773f23622 (after 0.47.04-r1) - does PRINT_MODE:TEXT work?

(If this is related to #1587, headless mode is intended to disable the graphical display, not the console.)

**Comment by lethosor** [comment]
Assuming this was resolved in 0.47.04-r2.
